sequencer.go

```diff
@@ -76,6 +76,19 @@ func (r *SeedSequencer) Rewind() {
 	r.current = 0
 }
 
+// RegenerateInvalidSeeds regenerates the index to match the unexpected seed content
+func (r *SeedSequencer) RegenerateInvalidSeeds(ctx context.Context, n int) error {
+	for _, s := range r.seeds {
+		if s.IsInvalid() {
+			if err := s.RegenerateIndex(ctx, n); err != nil {
+				return err
+			}
+			s.SetInvalid(false)
```

Comment:
oh, I see it's being done here. Either works but it might be better to do it in the seed since that's where the regenerate happens

Reply:
Yes, it's probably better to do that in `fileseed`. I moved it.